Source/Lib/Codec/EbEncHandle.c

```diff
@@ -2543,9 +2516,9 @@ static EB_ERRORTYPE VerifySettings(\
 		return_error = EB_ErrorBadParameter;
     }
 
-    if (config->tune > 2) {
-        SVT_LOG("SVT [Error]: Instance %u : Invalid Tune. Tune must be [0 - 2]\n", channelNumber + 1);
-        return_error = EB_ErrorBadParameter;
+    if (config->tune != 1) {
+        SVT_LOG("SVT [WARNING]: -tune is deprecated. Encoder always uses OQ - PSNR / SSIM optimized mode\n");
```

Comment:
I think we should stop at SVT [WARNING]: -tune is deprecated. 
otherwise it would feel like the encoder is not good for VMAF / visual

Reply:
sure, will do.